### DISCUSSION TITLE
Use a different temperature sensor?

### DISCUSSION THREAD

**Discussion by mandrean** [itavero/homebridge-z2m#625] [Q&A]
Hi,

I have a Zigbee floor thermostat available in HomeKit via `homebridge-z2m`

It can read both the ambient temperature in the bathroom, and the floor temperature from the floor sensor.

The physical display shows the floor temp, but in HomeKit I see the ambient room temp!

Is there any way I can switch this? I want to see the floor temp in HomeKit as well

**Comment by itavero** [comment]
Can you share which device it actually is?

**Reply by mandrean** [reply]
It's the Namron Zigbee floor-heating thermostat!

https://www.namron.com/products/namron-zigbee-touch-termostat-16a-hvit/
